db/warehouse/migrate/20251201163146_add_substance_use_and_supportive_services_to_scorecard.rb

```diff
@@ -0,0 +1,14 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+class AddSubstanceUseAndSupportiveServicesToScorecard < ActiveRecord::Migration[7.2]
+  def change
+    add_column :boston_project_scorecard_reports, :substance_use_treatment_service, :jsonb
+    add_column :boston_project_scorecard_reports, :supportive_services, :boolean, default: false
```

Comment:
should be non-nullable?
```suggestion
    add_column :boston_project_scorecard_reports, :supportive_services, :boolean, default: false, null: false
```